packages/nuxt/src/types.ts

```diff
@@ -30,13 +30,6 @@ export interface UnocssNuxtOptions extends UserConfig {
    */
   preflight?: boolean
 
-  /**
-   * Adjust the position of the `uno.css` injection. (Depends on `mode`)
-   *
-   * @default 'first'
-   */
-  injectPosition?: 'first' | 'last' | number | { after?: string }
```

Comment:
Removing this will be a breaking change, we should keep the type and do a console warning to say "We temporarily reverted the support for `injectPosition` while seeking for better solution. It's not effective at this moment."